src/components/common/CardRow/CardRow.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0'
+import styled from 'styled-components'
+
+import { GlobalStyles, ThemeToggler } from 'storybook/decorators'
+
+import { Card } from '../Card/index'
+import { CardRow, CardRowProps } from '.'
+
+export default {
+  title: 'Common/CardRow',
+  component: CardRow,
+  decorators: [GlobalStyles, ThemeToggler],
+} as Meta
+
+const CardBody = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  p {
+    margin-bottom: 0px;
+  }
+  h3 {
+    margin: 2px 3px -1px 3px;
+  }
+  span {
+    font-weight: bold;
+    font-size: 12px;
+    margin: 2px 1px 2px 0px;
+    &.red {
+      color: red !important;
+    }
+    &.green {
+      color: #06b506 !important;
```

Comment:
#00D897 (use this green)

Reply:
Or better yet, take it from the theme, see how it's done in other components https://github.com/gnosis/gp-ui/blob/4da5595b204f6dc2b59bf4bb61c809cb37221ece/src/components/orders/OrderSurplusDisplay/index.tsx#L21